src/main/webapp/adminOnly/AdminOnly.js

```diff
@@ -81,13 +81,9 @@ export const AdminOnly = hh(class AdminOnly extends Component {
   };
 
   isCurrentUserAdmin() {
-    User.getUserSession(this.props.userSessionUrl).then(
-      resp => {
-        this.setState({ isORSP: resp.data.isORSP });
-      }
-    ).catch(error => {
-      this.setState(() => { throw error; });
-    });
+    isCurrentUserAdmin().then(response => {
```

Comment:
This naming is confusing ... the method name here is the same as the method it is in ... so it looks like a recursive call, but maybe it isn't?

Reply:
it is, now that I look at it carefully. Eitherway I removed all these methods everywhere in realtion to another [comment](https://github.com/broadinstitute/orsp-pub/pull/178#discussion_r293786287) below this PR